source/MaterialXRenderGlsl/GlslProgram.h

```diff
@@ -102,6 +102,8 @@ class MX_RENDERGLSL_API GlslProgram
         string path;
         /// Unit
         string unit;
+        /// Colorspace 
+        string colorspace;
```

Comment:
For GLSL program reflection cache colorspace which is now available for query w/o going back to the original MTLX elements.